test/utils/asserts.hpp

```diff
@@ -15,4 +15,5 @@
     catch (...)                                                                              \
     {                                                                                        \
         ADD_FAILURE() << "Unexpected exception type thrown.";                                \
-    }
+    }                                                                                        \
+    (void)0
```

Comment:
what does this do?

Reply:
Ensure you need to add a trailing semicolon.